src/components/helloWorld.ts

```diff
@@ -0,0 +1,7 @@
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: 'HelloWorld',
+  template: 'hello world',
+  expose: [],
+})
```

Comment:
are you planning on shipping this?

Reply:
I think we'll need something like this for our unit tests. We should probably make a folder or something for files that are just for tests. Thoughts?

Reply:
yeah, I guess it's no more offensive than `@/utilities/testHelpers`

Reply:
right, I just couldn't put this there. Well I guess I could have as the default export but that feels weird. I vote we keep moving as as we have more stuff like this maybe we find a spot for this stuff. We already wanna organize the repo more at some point.